.github/workflows/ci-build.yml

```diff
@@ -15,15 +15,19 @@ jobs:
           sudo apt-get -y install lua${{ matrix.lua_version }}
           sudo apt-get -y install liblua${{ matrix.lua_version }}
           sudo apt-get -y install luarocks
+          sudo luarocks install luacheck
+          sudo luarocks install luacov
+          sudo luarocks install luacov-coveralls
           sudo luarocks install luaunit
           sudo luarocks install mockagne
+          sudo luarocks install luasocket
```

Comment:
Can you combine the separate calls into a single call? Same for apt-get install.

Reply:
Done via `for` loop. The tool itself does unfortunately not have this option.